synapse/models/risk.py

```diff
@@ -1022,6 +1038,27 @@ def getModelDefs(self):
 
                 )),
 
+                ('risk:outage:cause:taxonomy', {}, ()),
+                ('risk:outage', {}, (
```

Comment:
Do we want a :type prop to capture the type of outage?

Reply:
"type" of outage is not specific enough. i created the `:cause` property as a more specific / analytically relevant taxonomy.